src/modules/launcher/Wox.Plugin/Wox.Plugin.csproj

```diff
@@ -77,6 +77,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\common\ManagedCommon\ManagedCommon.csproj" />
```

Comment:
if we're moving in manage common, we need to test out the installer works here as well.

Reply:
Already built the MSI and tested installation: ManagedCommon.dll is deployed in `C:\Program Files\PowerToys\modules\launcher`